packages/yarnpkg-core/sources/StreamReport.ts

```diff
@@ -169,11 +171,12 @@ export class StreamReport extends Report {
     lastTitle?: string;
   }> = new Map();
 
+  private readonly enableProgressBars: boolean;
```

Comment:
We don't need a new value; we can just define `progressStyle === null` as being "no progress bars"